Library/Formula/coturn.rb

```diff
@@ -0,0 +1,22 @@
+class Coturn < Formula
+  desc "Free open source implementation of TURN and STUN Server"
+  homepage "https://github.com/coturn/coturn"
+  url "http://turnserver.open-sys.org/downloads/v4.4.5.4/turnserver-4.4.5.4.tar.gz"
+  sha256 "ca6d23d5d16011869029054d937c2b46f3a657e6be8c7720cb946dddd9de9a99"
+
+  depends_on "libevent"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--mandir=#{man}",
+                          "--prefix=#{prefix}"
+
+    system "make", "install"
+  end
+
+  test do
+    system "turnadmin", "-l"
```

Comment:
This needs to be `system bin/"turnadmin", "-l"`, presuming the binary is in `bin`. We use full paths in testing.